pages/highlights/templates/highlights/highlight_detail.html

```diff
@@ -0,0 +1,147 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+<!-- Main Content -->
+<div class="py-8">
+    <!-- Announcement (if present) -->
+    {% if highlight.announcement %}
+        <div class="announcement">
+            {% if highlight.rendered_announcement %}
+                {{ highlight.rendered_announcement|safe }}
+            {% else %}
+                {{ highlight.announcement }}
+            {% endif %}
+        </div>
+    {% endif %}
+    
+    <!-- Title and Publication Date -->
+    <h1 class="text-3xl text-pp-almost-black mb-2">
+        {{ highlight.title }}
+    </h1>
+    <p class="text-pp-dark-grey mb-6">
+        Published: {{ highlight.created_at|date:"F j, Y" }}
+    </p>
+    
+    <!-- Featured Image (floated right on medium+ screens) -->
+    {% if highlight.featured_image %}
+        <figure class="figure-float">
+            <img src="{{ highlight.featured_image.url }}" 
+                 alt="{{ highlight.title }}">
+            {% if highlight.figure_caption %}
+                <figcaption>
+                    {{ highlight.figure_caption }}
+                </figcaption>
+            {% endif %}
+        </figure>
+    {% endif %}
+    
+    <!-- Main Content -->
+    {% if highlight.content %}
+        <div class="prose prose-lg max-w-none text-pp-dark-grey mb-8">
+            {% if highlight.rendered_content %}
```

Comment:
I forgot to ask about this, why have this`if` block ? Will there be a scenario where there is no `rendered_content` ? 🤔